tests/test_problems.py

```diff
@@ -0,0 +1,53 @@
+import sqlite3
+
+import pytest
+
+from sirenity.euler import ProblemManager
+from sirenity.euler.problem import Problem
+
+# Clear db
+with open("test.db", "w") as f:
+    f.write("")
+
+manager = ProblemManager("test.db")
+
+
+class TestClass:
+    """Tests ProblemManager"""
+
+    def test_create_table(self) -> None:
+        """Tests that table is created"""
+        con = sqlite3.connect(manager._file)
+        cur = con.cursor()
+        manager.create_table()
+        cur.execute(
+            "SELECT name FROM sqlite_master WHERE type='table' AND name='problems';"
+        )
+        assert len(cur.fetchall()) == 1
+        con.close()
+
+    def test_add_and_get_problem(self) -> None:
+        """Tests that problems are added and are retrieved"""
+        manager.add_to_db(prompt="Problem prompt", solution="solution")
+        problem = manager.get_at_id(1)
+        assert problem.id == 1
+        assert problem.prompt == "Problem prompt"
+        assert problem.solution == "solution"
```

Comment:
```suggestion
        self.assertEqual(problem.id, 1)
        self.assertEqual(problem.prompt, "Problem prompt")
        self.assertEqual(problem.solution, "solution")
```

The unittest library has functions for these

Reply:
I though we were using pytest

Reply:
We are, pytest works with the unittest library

Reply:
Oh apologies, let me rethink this

Reply:
Yea, I'm happy using the `unittest` library right now and use pytest  to run tests and get coverage stats. But we might be able to drop pytest if something else can do coverage. We'll have to consider if I want it to keep pytest and work out what the advantages are of keeping it.